### PR TITLE
Add description in carousel of blocks

### PR DESCRIPTION
**Pull request by fkroockmann**
https://github.com/backbee/backbee-standard/issues/127

### CODE REVIEW

File: src/tb/apps/content/views/content.view.contribution.index.js [hunk 1]
```diff
@@ -189,7 +199,32 @@ define(
                     html = html + Renderer.render(carouselBlocksTpl, data);
                 }
 
-                jQuery(this.carouselBlocksId).html(html);
+                carousel.html(html);
+
+                carousel.find(this.carouselBlockClass).on('click', this.onBlockClick);
+            },
+
+            onBlockClick: function (event) {
```
Comment (ndufreche) [inline src/tb/apps/content/views/content.view.contribution.index.js]: This have to be done with hover i think, not at the click and with a tooltip but with not a popin

**Comment by ndufreche** [comment]
https://jqueryui.com/tooltip/

**Comment by fkroockmann** [comment]
Currently, it's done like that...

**Comment by ndufreche** [comment]
Yeah, but is not a good solution and we can integrate a html content in a tooltip
see => http://jsfiddle.net/Aa5nK/12/

**Comment by mickaelandrieu** [comment]
@ndufreche we have a lot of popins, why introduce "tooltips" ?

@clementbonfils decide please: tooltip or popin ?

**Comment by ndufreche** [comment]
@mickaelandrieu because in this case a tooltip it's the exact behavior we need. A popin's behavior implies an interaction, here there no interactions, just an description.

**Comment by fkroockmann** [comment]
A jQuery tooltip accept just a text, we need a picture, a description and a type...

**Comment by ndufreche** [comment]
see => http://jsfiddle.net/Aa5nK/12/

**Comment by fkroockmann** [comment]
it's not html, its a text..

**Comment by ndufreche** [comment]
In the jsfiddle example it's HTML

<a href="#" title="<span class='red'>That's what this widget is</span>">Tooltips</a>
=> <span class='red'>That's what this widget is</span>

**Comment by mickaelandrieu** [comment]
@ndufreche I'm not sure this is valid HTML to put HTML content inside ``title`` attribute (http://www.w3.org/TR/html401/struct/global.html#h-7.4.3).

ping @clementbonfils : decide please as it's more business/functional issue than technical's one :)

**Comment by crouillon** [comment]
Ok for the popin in my opinion
But for next releases let's think about tooltip or even nothing (the description should be available yet from the block display popin, it seems to be enough)

**Comment by ndufreche** [comment]
Ok, I merged it.

**Pull request merged**